source/manual/manage-ruby-dependencies.html.md

```diff
@@ -5,29 +5,42 @@ description: How we manage our Ruby dependencies using Dependabot, who can merge
 section: Dependencies
 layout: manual_layout
 parent: "/manual.html"
-last_reviewed_on: 2019-04-17
+last_reviewed_on: 2019-11-04
 review_in: 6 months
 ---
 
 We're [obliged to keep our software current][current].
 
 To help with this, we use a service called [Dependabot][] to perform automated dependency upgrades.
 
-## Add Dependabot to a repo
+## Dependabot for new dependency versions
+
+### Add Dependabot to a repo
 
 1. Give Dependabot [access to the repo][access] (only GitHub org owners can do this)
-2. Go to [Dependabot admin][admin] and click "Add project"
+1. Go to [Dependabot admin][admin] and click "Add project"
 
-## Ask Dependabot to bump dependencies
+### Ask Dependabot to bump dependencies
 
 By default Dependabot will bump dependencies once a day, but you can ask it to bump manually:
 
 Go to [Dependabot admin][admin] and click "Bump now" for your project
 
-## Security
+### Security
 
 There are 2 safeguards to prevent unauthorised code changes. Firstly, Dependabot can only update the repositories that we [explicitly allow on GitHub][access]. This prevents code changes to other repos. Secondly, we've [set up branch protection](/manual/configure-github-repo.html#auto-configuration) for all repos with the `govuk` label. This prevents Dependabot from writing directly to master.
 
+## Dependabot for security vulnerabilities in dependencies
+
+Since GitHub acquired Dependabot, there are two ways that it updates
+dependencies in your project. Security vulnerabilities come through
+the GitHub dependabot channel under the username `dependabot-preview`.
+
+### Enable "automated security fixes" in the GitHub UI for your repo
+
+1. Go to Security settings for your GitHub repo
+1. Click the "Off: Automated Security Fixes" dropdown and change it to "On"
```

Comment:
I don't quite get the relevance of these instructions. Is this something that should be done for all repositories, and what's the effect if they're followed?

Reply:
Maybe we should just link here: https://help.github.com/en/github/managing-security-vulnerabilities/configuring-automated-security-fixes.

Reply:
On reflection, I backed out these changes. It's now simple date bump. Maybe in six months GitHub would have integrated Dependabot more and there won't be two separate workflows?

Reply:
Ok :+1: